resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json

```diff
@@ -25,7 +25,7 @@
             {{ autoimport }}
             {{ xattrs }}
             {{ no_conflicts }}
-            "revs_limit": 20,
+            "revs_limit": 15,
```

Comment:
this will fail SGW start if allow_conflicts is False. Any reason why you need 20?